data-files/samples/targets.Status.Any

```diff
@@ -0,0 +1,12 @@
+{ 
+    currentUser = "Sample User"; 
+    sessions = ( "1target", "2targets", "5targets", "moving", "jumping", "mixedtypes"); 
+    settingsVersion = 1; 
+    users = ( 
+        { 
+            completedSessions = ( ); 
```

Comment:
As a general comment, once this `completedSession` list is full this sample will just show "All Sessions Completed". I assume this is fine, but right now we don't really have a good mechanism for clearing out this array...

Reply:
I agree this would be a practical problem for trying out the samples, but I don't really have a good solution for this. They can always revert the file 🤷‍♂️